src/routine.cpp

```diff
@@ -24,6 +24,7 @@ namespace clblast {
 // The constructor does all heavy work, errors are returned as exceptions
 Routine::Routine(Queue &queue, EventPointer event, const std::string &name,
                  const std::vector<std::string> &routines, const Precision precision,
+                 std::initializer_list<const char *> source_pct,
```

Comment:
This is done to allow using built-in pad/copy/transpose code together with a custom main kernel, calling the now-exported `PadCopyTransposeMatrix<T>()` from the custom host code. Unless there is a misunderstanding, I *do* want to have this split.